1-org/envs/shared/org_policy.tf

```diff
@@ -90,6 +90,14 @@ module "restrict_protocol_fowarding" {
   IAM
 *******************************************/
 
+// Time sleep
+resource "time_sleep" "wait_org_policies" {
```

Comment:
Can we improve the name for this resource?
With `wait_org_policies` I understand that this resource is waiting for an org policy creation, but actually it is waiting for `logs_export` module creation instead.

Another improve here would be to improve the comment. `// Time sleep` doesn't tell us much more than the resource's name `time_sleep` - I believe we should add the reason of this `time_sleep` existence in the comment instead.